src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -242,14 +242,18 @@ export const TradeInput = memo(() => {
     [isSnapshotApiQueriesPending, votingPower],
   )
 
-  const { sellAssetAccountId, buyAssetAccountId, setSellAssetAccountId, setBuyAssetAccountId } =
-    useAccountIds()
+  const {
+    sellAssetAccountId: initialSellAssetAccountId,
+    buyAssetAccountId: initialBuyAssetAccountId,
```

Comment:
Technically no only defaults, but making it clear that the *main* intent of these is to have initial values - else if all we cared about was selecting the buy/sell AccountIds, we could directly use the selectors that are consumed there